README.md

```diff
@@ -6,3 +6,23 @@ The UI is being designed on Figma if you'd like to view
 
 - [Figma UI Prototype](https://www.figma.com/file/i7IjqvJVL6c5h2Tdzuul3c/Moo-Twitter-Design-File)
 - [Figma Discussion Jam Board](https://www.figma.com/file/onuHbJL39i2be0OosK4vYX/Moo-Twitter-Discussion-Board?node-id=0%3A1)
+
+### To run the application :
+
+- Clone / Download :
+
+       git clone https://github.com/your_username_/Project-Name.git
```

Comment:
Hi @Jassi10000 you can give link to this repository

Reply:
@shivaylamba Yeah , but every user will have its own username , so link will also be different for everyone
Wouldn't it be better if they fork and then themselves get the link 
Your views on this  ?